components/Buttons/src/MDCButton.h

```diff
@@ -47,6 +47,16 @@
  */
 @property(nonatomic, assign) CGFloat inkMaxRippleRadius UI_APPEARANCE_SELECTOR;
 
+/**
+ This property determines if an @c MDCButton should use the legacy ripple behavior or not.
```

Comment:
Tip:  I think you mean "MDCInkView behavior" and not "legacy ripple. Naming is confusing, naming is hard, 3 ripple/inks is absurd.